CloudScrape.cs

```diff
@@ -0,0 +1,588 @@
+        {
+            client = new CloudScrapeClient(apiKey, accountId);
+        }
+
+        /// <summary>
+        /// Default Client
+        /// </summary>
+        /// <returns></returns>
+        public CloudScrapeClient DefaultClient()
+        {
+            CheckState();
+
+            return client;
+        }
+
+        /// <summary>
+        /// Execute Method
+        /// </summary>
+        /// <returns></returns>
+        public CloudScrapeExecutions Executions()
+        {
+            CheckState();
+
+            return client.Executions();
+        }
+
+        /// <summary>
+        /// Run Method
+        /// </summary>
+        /// <returns></returns>
+        public CloudScrapeRuns Runs()
+        {
+            CheckState();
+
+            return client.Runs();
+        }
+
+        /// <summary>
+        /// Check State
+        /// </summary>
+        private void CheckState()
+        {
+            if (client == null)
+            {
+                throw new Exception("You must call init first before using the API");
+            }
+        }
+    }
+
+    /// <summary>
+    /// Cloud Scrape Client
+    /// </summary>
+    class CloudScrapeClient
+    {
+        private string apiKey;
+        private string accountId;
+        private string accessKey;
+
+        private string _endPoint = "https://app.cloudscrape.com/api/";
+        private string _userAgent = "CS-ASP-CLIENT/1.0";
+        private int _requestTimeout = 3600;
+
+        private CloudScrapeExecutions objExecutions;
+        private CloudScrapeRuns objRuns;
+
+        public string EndPoint
+        {
+            get { return _endPoint; }
+            set { _endPoint = value; }
+        }
+
+        public string UserAgent
+        {
+            get { return _userAgent; }
+            set { _userAgent = value; }
+        }
+        
+        public int RequestTimeout
+        {
+            get { return _requestTimeout; }
+            set { _requestTimeout = value; }
+        }
+
+        
+
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="apiKey"></param>
+        /// <param name="accountId"></param>
+        public CloudScrapeClient(string apiKey, string accountId)
+        {
+            this.apiKey = apiKey;
+            this.accountId = accountId;
+            this.objExecutions = new CloudScrapeExecutions(this);
+            this.objRuns = new CloudScrapeRuns(this);
+        }
+
+        /// <summary>
+        /// Make a call to the CloudScrape API
+        /// </summary>
+        /// <param name="url"></param>
+        /// <param name="method"></param>
+        /// <param name="body"></param>
+        /// <returns></returns>
+        public CloudRespose Request(string url, string method = "GET", string body = null)
+        {
+            string userPassword = CreateMD5(accountId + apiKey).ToLower();
+
+            var req = System.Net.HttpWebRequest.Create(EndPoint + url) as HttpWebRequest;
```

Comment:
Is there any request timeouts that have a low default ? Some of the requests might take long by design since it'll wait for an execution to finish.